lib/methods/dynamics/webApi/read.js

```diff
@@ -97,8 +97,8 @@ module.exports = (
   /**
    * Fetches all accounts for the given accountIds
    *
-   * @param {Array} accountIds
-   * @return {Promise<Array>}
+   * @param {Array} accountIds An array of accountIds
+   * @return {Promise<[Account]>} An array of Accounts defined by the input array of Ids
```

Comment:
Is this the syntax for an array of accounts? Usually you see `Account[]` or `Array<Account>`